### ISSUE TITLE
Bug report: Application icon does not display properly on Ubuntu

### ISSUE THREAD

**Issue by viperperidot** [monero-project/monero-gui#3132]
I have installed the latest version of the Monero GUI v0.17.0.1 on Ubuntu version 20.04.1. In the application tray the Monero icon does not display properly. Here is a screenshot of what it looks like: https://imgur.com/PTroRVp

The application does appear correctly on the dock once Monero GUI is launched and running.

I checked the .desktop file and under icon it just says 'monero'. I tried replacing this field with a file path to the Monero logo and that fixed the problem, however once you launch the application any changes made to the .desktop file get overwritten and the application icon does not display properly once again.

Any suggestions on how to fix this would be appreciated, thanks.

**Comment by haluzpav** [comment]
Yea, I see there were several attempts, starting from #2292, none of them got merged. :disappointed:

**Comment by selsta** [comment]
@haluzpav https://github.com/monero-project/monero-gui/pull/3251

On first startup it should ask you if you want to install desktop file.

**Comment by haluzpav** [comment]
@selsta Yes, it does, but it shows only a generic icon, as in OP's screenshot. There's no Monero icon under `.local/share/icons/` where the most of other apps put their icons. I have Fedora with Gnome, so maybe it's somehow related to Gnome or its themes, if you believe there should be the icon? However I don't see where could it take the icon from.
